### PR TITLE
Add more boilerplate

### PR DESCRIPTION
**Pull request by BerkOzdilek**
Add more boilerplate

### CODE REVIEW

File: src/main/java/com/heycar/pairingsession/controller/ListingsController.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+public class ListingsController {
+
+    private final ListingsService listingsService;
+
+    public ListingsController(ListingsService listingsService) {
+        this.listingsService = listingsService;
+    }
+
+    /** Should accept the following request:
+     curl -X POST 'http://localhost:8080/listings/{dealerId}/' \
```
Comment (grahamhunter-heycar) [inline src/main/java/com/heycar/pairingsession/controller/ListingsController.java]: Do we want to give the url to them, for me its interesting to see if they come up with one that has good REST principles

File: src/main/java/com/heycar/pairingsession/controller/ListingsController.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
+package com.heycar.pairingsession.controller;
+
+import com.heycar.pairingsession.service.ListingsService;
```
Comment (grahamhunter-heycar) [inline src/main/java/com/heycar/pairingsession/controller/ListingsController.java]: I would like see them come up with the package structure themselves, maybe put all the classes together in the pairsession package so they can move them

File: src/main/java/com/heycar/pairingsession/controller/ListingsController.java (hunk 1 above)
Comment (BerkOzdilek) [reply src/main/java/com/heycar/pairingsession/controller/ListingsController.java]: hmm yea makes sense! will update it

File: src/main/java/com/heycar/pairingsession/controller/ListingsController.java (hunk 2 above)
Comment (BerkOzdilek) [reply src/main/java/com/heycar/pairingsession/controller/ListingsController.java]: 👍  will do

File: src/main/java/com/heycar/pairingsession/Listing.java [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+import javax.persistence.Entity;
+import javax.persistence.Id;
+
+@Entity
+public class Listing {
+
+    @Id
+    private String id;
+    @Column
+    private String dealerId;
```
Comment (paruir-dadaian) [inline src/main/java/com/heycar/pairingsession/Listing.java:13]: I would remove this column to not give hint. It's interesting to see how candidate makes decision
to create or not create entity for dealer

File: src/main/java/com/heycar/pairingsession/Listing.java [hunk 4]
```diff
@@ -0,0 +1,91 @@
+package com.heycar.pairingsession;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+
+@Entity
```
Comment (ggonchar) [inline src/main/java/com/heycar/pairingsession/Listing.java:7]: @BerkOzdilek @grahamhunter-heycar @paruir wdyt about the persistence to be open for a discussion comparing to precooking the choice with ORM?

**Pull request closed without merging**